libs/output-mapping/tests/DeferredTasks/Metadata/ColumnMetadataTest.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\OutputMapping\Tests\DeferredTasks\Metadata;
+
+use Generator;
+use Keboola\OutputMapping\DeferredTasks\Metadata\ColumnMetadata;
+use Keboola\StorageApi\Metadata;
+use Keboola\StorageApi\Options\Metadata\TableMetadataUpdateOptions;
+use PHPUnit\Framework\TestCase;
+
+class ColumnMetadataTest extends TestCase
+{
+    private const TEST_TABLE_ID = 'in.c-testApply.table';
+    private const TEST_PROVIDER = 'keboola.sample-component';
+    private const TEST_METADATA = [
+        'id' => [
+            [
+                'key' => 'timestamp',
+                'value' => '1674226231',
+            ],
+        ],
+        'aa_caa' => [
+            [
+                'key' => 'KBC.datatype.basetype',
+                'value' => 'STRING',
+            ],
+            [
+                'key' => '1',
+                'value' => '',
+            ],
+        ],
+    ];
+
+    public function applyDataProvider(): Generator
+    {
+        yield 'load at once' => [
```

Comment:
toto je trošku misleading, protože to akorát nastaví chunk na 100 - resp. misleading je to bulkSize=null, dal bych tam třeba bulkSize = 10 a na ten null se úplně vyprdl, čímž se zbavíme i podmínky https://github.com/keboola/platform-libraries/pull/51/files#diff-37bca47ab606e153792ba32891886b654ece99c084ff7cac37be22a88ad86257R120 která se mě ani trošku nelíbí, protože nemám rád podmínky v testech :)

Reply:
fixed